make/lint.mk

```
@@ -11,5 +11,7 @@ lint-yaml: ${YAML_FILES}
 .PHONY: lint-go-code
 ## Checks the code with golangci-lint
 lint-go-code: generate
-	$(Q)go get github.com/golangci/golangci-lint/cmd/golangci-lint
-	$(Q)${GOPATH}/bin/golangci-lint ${V_FLAG} run
\ No newline at end of file
+ ifeq (, $(shell which golangci-lint))
```

This will make the output more clear
```suggestion
 ifeq (, $(shell which golangci-lint 2>/dev/null))
```

---

thanks! done in e9c760c